Tests/MongoSwiftTests/ReadWriteConcernTests.swift

```diff
@@ -88,11 +88,11 @@ final class ReadWriteConcernTests: MongoSwiftTestCase {
             expect(client.readConcern).to(beNil())
 
             // expect that a DB created from this client inherits its unset RC 
-            let db1 = try client.db("test")
+            let db1 = try client.db(type(of: self).testDatabase())
```

Comment:
my bad on the unhelpful names 😛

Reply:
@kmahar No worries! I've made far worse variable names before :).

@patrickfreed I may be looking at the wrong diff, but it seems like this hasn't been done yet?

Reply:
I think we could just leave them for now - as you said it's outside the scope of this ticket, and I think we can figure out what they do easily enough from the comments

Reply:
Yeah I thought you meant something like "think about doing this in a separate ticket" by your comment, so I didn't do anything here.

Reply:
Sure, sounds fine to me. I think this is a trivial enough change (since its just a set of renames) that we _could_ do it here if we want. Doing it outside of this ticket is fine as well, lets at least file the ticket!